scripts/fetch

```diff
@@ -9,22 +9,22 @@ cores=$(nproc)
 manifest_dir="${MANIFEST_DIR?}"
 manifest_repo="${MANIFEST_REPO?}"
 
-mkdir -p "${base_dir}" "${manifest_dir}"
+mkdir --parents "${base_dir}" "${manifest_dir}"
```

Comment:
`mkdir -p` is very widely known. I'd have to look up what `--parents` means

Reply:
@daurnimator I understand your point. The question is if we should always go for the long variant for consistency or not. I am open to both in this case.

Reply:
I'd argue consistency here hurts understandability.

Reply:
Anyone that does not know what --parents means got to be one of todays lucky 10,000. I don't mind the idea of being verbose in code, and sticking to shorthand day to day.

Makes it easier for people new to this stuff.